app/views/admin/poll/polls/show.html.erb

```diff
@@ -3,6 +3,5 @@
 <div id="poll-resources">
   <%= render "subnav" %>
 
-  <%= render "search_questions" %>
```

Comment:
I think this was the only usage of `search_questions`, if we're not planning on using it in the near future there is things pending to be cleaned up:

/app/views/admin/poll/polls/search_questions.js.erb
/app/views/admin/poll/polls/_search_questions.html.erb
/app/views/admin/poll/polls/_search_questions_results.html.erb

https://github.com/consul/consul/blob/polls_minor_changes/config/routes.rb#L276
https://github.com/consul/consul/blob/polls_minor_changes/app/models/abilities/administrator.rb#L59
https://github.com/consul/consul/blob/polls_minor_changes/app/controllers/admin/poll/polls_controller.rb#L4
https://github.com/consul/consul/blob/polls_minor_changes/app/controllers/admin/poll/polls_controller.rb#L50-L55